### ISSUE TITLE
Admin error log

### ISSUE THREAD

**Issue by wads24** [ForumHulp/Delete-Inactive-Users#9]
I'm seeing this error many times in the admin error log.  
ActionError while creating image
» Error in [ROOT]/ext/forumhulp/deleteinactiveusers/cron/task/core/delete_inactive_users.php on line 65: in_array() expects parameter 2 to be array, null given

**Comment by wads24** [comment]
Bump?  Does anyone read these?

**Comment by ForumHulp** [comment]
You have to give me more information as i can not reproduce this.
For example what's in delete_inactive_users_warning in config_txt table

**Comment by wads24** [comment]
a:1:{i:0;s:0:"";}

**Comment by wads24** [comment]
My error log in phone is just fill if these errors.  http://pichost.org/images/2018/11/20/Screenshot_20181119-230909_Chrome.jpg

**Comment by ForumHulp** [comment]
Can you replace it with [[]] and test

**Comment by wads24** [comment]
I think the errors have stopped.  Reminder email is being sent every hour...  How can I change that

**Comment by wads24** [comment]
Also..  how can I register on your forum?  I emailed you about errors

**Comment by wads24** [comment]
I changed send reminders to no and emails are still being sent

**Comment by wads24** [comment]
Yes, it is sending an email every hour.  I have a test account that is inactive and got an email every hour

**Comment by ForumHulp** [comment]
There is really something wrong on your server.
This cron only runs once a day, not every hour. Look at filepermissions for example

**Comment by wads24** [comment]
What should file permissions be set for?  What specific files should I look at

**Comment by ForumHulp** [comment]
In cache it should be 775 minimum.

**Comment by wads24** [comment]
Yes permission in cache is 755

**Comment by wads24** [comment]
Still no go...   Also now it's blocking forum emails after upgrade to 3.2.4

**Comment by ForumHulp** [comment]
Try to install again
